client/jest.config.js

```diff
@@ -0,0 +1,11 @@
+const { createDefaultPreset } = require('ts-jest');
```

Comment:
tohle by melo byt spise
```
import { createDefaultPreset } from 'ts-jest';
```

jinak to Eslint oznacuje jako chybu a nejdou mi spustit testy :)